blockchain/blockchain.go

```diff
@@ -40,6 +46,9 @@ type Blockchain struct {
 	ChainId             eos.Checksum256
 	PlatformAccountName string
 	sponsorUrl          string
+	optsMutex           sync.Mutex
```

Comment:
why did you add mutex?

Reply:
because GetTxOpts can be invoked from different threads.